### PR TITLE
Imp various fixes improvements

### PR DESCRIPTION
**Pull request by flotho**
Hi @robertdebock 
Small proposal here : 
* force the cron installation 
* don't use creates in the command executipon so that new domains could be added at each execution. Otherwise it won't be run https://docs.ansible.com/ansible/latest/collections/ansible/builtin/command_module.html#parameter-creates

### CODE REVIEW

**Comment by flotho** [comment]
Hi @robertdebock , any chance to hae a review on this one ?
Regards

**Comment by robertdebock** [comment]
Okay, tests will run in a bit. I've just pushed many small changes to all roles, so CI will like start in several hours. I'll try to keep an eye on it.

File: vars/main.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@
 # vars file for certbot
 certbot_package:
   - certbot
+  - cron
```
Comment (robertdebock) [inline vars/main.yml:5]: Cron is called differently for different distributions, you can make something like this:

```yaml
_certbot_packages:
  default:
    - certbot
    - cronie
  Alpine:
    - certbot
    - dcron
  Debian:
    - certbot
    - cron

certbot_packages: "{{ _certbot_packages[ansible_os_family] | default(_certbot_packages['default']) }}"
```

**Review by robertdebock — COMMENTED** [review]
Hey, please review the 1 item that makes CI fail: `cron`.

File: vars/main.yml (hunk 1 above)
Comment (flotho) [reply vars/main.yml]: thanks @robertdebock  , PR is updated,
Let me know

**Comment by flotho** [comment]
Hi @robertdebock , changes done!

**Comment by flotho** [comment]
FIX #6

**Comment by robertdebock** [comment]
LGTM, merged. Thanks I'll review the CI tests and fix if required. Good job, thanks a lot!

**Comment by flotho** [comment]
you're welcome
Your work is awesome !

**Pull request merged**